man/subscription-manager.8

```diff
@@ -604,8 +604,8 @@ option
 .B --remove=NAME
 Removes a named override from the repositories specified with the
 .B --repo
-option
-
+option.  Overrides inherited from environments the system belongs in will not be removed. They can only be removed by either: removing them from the environment they originate from or removing the system from those environments.
+The source will not be removed if its source is one of the environment(s) the system belongs in. It can only be removed by either: removing the override from the environment it originates from or removing the system from the environment the override originates from.
```

Comment:
I think you added the text I proposed for the `--remove-all` command into the `--remove` command. If you check my other comment, it was done against line 610 (where the --remove-all command description is)